### ISSUE TITLE
Failed to access tables on alibaba oss via spark + gluten + velox

### ISSUE THREAD

**Issue by leesf** [oap-project/gluten#1366]
**Describe the bug**
refer to https://github.com/facebookincubator/velox/issues/4615

**To Reproduce**

**Expected behavior**

**Additional context**

**Comment by FelixYBW** [comment]
@Yohahaha Is OSS supported added to velox?

**Comment by Yohahaha** [comment]
> @Yohahaha Is OSS supported added to velox?

No, OSS support feature is partially enabled in Alibaba could EMR 5.11.1, we have not push this feature to Velox due to some internal dependency has not been opensourced, but it will done in soon.

@leesf you could contract us offline to get OSS support for test.

**Comment by leesf** [comment]
thanks, would you please add my wechat: xleesf

**Comment by Tomccat3** [comment]
> > @Yohahaha Is OSS supported added to velox?
> 
> No, OSS support feature is partially enabled in Alibaba could EMR 5.11.1, we have not push this feature to Velox due to some internal dependency has not been opensourced, but it will done in soon.
> 
> @leesf you could contract us offline to get OSS support for test.

hi, i have the same question, how can we contract offline? my wechat: jigezhuanxiang
